<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qMasterPassword</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnAddUser">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/user_add.png</normaloff>:/icons/user_add.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDeleteUser">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/user_delete.png</normaloff>:/icons/user_delete.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmbUserName">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumContentsLength">
         <number>16</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtPassword">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLoginLogout">
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Categories</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>676</width>
             <height>491</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnLoginLogout</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>loginLogoutClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>592</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAddUser</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addUser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>713</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDeleteUser</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>deleteUser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>689</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtPassword</sender>
   <signal>returnPressed()</signal>
   <receiver>btnLoginLogout</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>422</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>566</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loginLogoutClicked()</slot>
  <slot>addUser()</slot>
  <slot>deleteUser()</slot>
 </slots>
</ui>
